src/label/Label.ts

```diff
@@ -45,12 +44,6 @@ export const LabelBase = ThemeableMixin(WidgetBase);
 
 @theme(baseCss)
 export default class Label extends LabelBase<LabelProperties>  {
-	@onPropertiesChanged()
-	protected onPropertiesChanged(event: PropertiesChangeEvent<this, LabelProperties>) {
```

Comment:
Ah that's a reasonable point, it should use the registry mixin if that's the case (there were zero tests covering this though) or at least none failed...